backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/rest/dto/VersionFileDto.kt

```diff
@@ -2,18 +2,20 @@ package com.tngtech.apicenter.backend.connector.rest.dto
 
 import com.fasterxml.jackson.annotation.JsonCreator
 import com.tngtech.apicenter.backend.domain.entity.ApiLanguage
+import com.tngtech.apicenter.backend.domain.entity.ServiceId
 
-data class SpecificationMetaData constructor(
+data class VersionMetaData constructor(
```

Comment:
I'm not sure what we gain by having this as a separate class. Could you please help me there? :)

Reply:
The converters to/from the `Version` domain object aren't "symmetric", there's a `VersionDto` (simply content + metadata) and a `VersionFileDto` (from the frontend upload form, whose file content needs to be parsed etc.) Both use the metadata class.

Reply:
Yes, that makes sense. Thanks :)